kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/installation/kroxylicious/CertManager.java

```diff
@@ -42,9 +44,11 @@ public CertManager() throws IOException {
     }
 
     private boolean isDeployed() {
-        List<GenericKubernetesResource> certManagerList = kubeClient().getClient().genericKubernetesResources("apiextensions.k8s.io/v1", "CustomResourceDefinition")
-                .list().getItems().stream().filter(crd -> crd.getMetadata().getLabels().get("app").equalsIgnoreCase("cert-manager")).toList();
-        return !certManagerList.isEmpty();
+        List<Deployment> deployments = kubeClient().getClient().apps().deployments().inAnyNamespace().list().getItems().stream()
+                .filter(deployment -> deployment.getMetadata().getLabels().get("app") != null
+                        && deployment.getMetadata().getLabels().get("app").equalsIgnoreCase("cert-manager")).toList();
```

Comment:
Looking for a CRD is superior to looking for labels on the deployment.  CRDs are part of the public API so much less likely to change, labels and names of deployments are implementation detail.

Reply:
That said, I'll approve so we can move on.